tests/ast/data/functions1.test

```diff
@@ -16,7 +16,7 @@ df05 = df.select(column("X", "A"))
 
```

Comment:
just to confirm, did you run update-expectations?

Reply:
AST merge gate seems to fail

Reply:
I missed it.